### PR TITLE
SDKCF-4287: Add simple logging functions

### PR DESCRIPTION
**Pull request by malayli**

### CODE REVIEW

File: Sources/RLogger/RLogger.swift [hunk 1]
```diff
@@ -31,7 +31,32 @@ public struct RLogger {
          default: ()
          }
          return message
-     }
+    }
+
+    @discardableResult
+    public static func verbose(message: String) -> String? {
```
Comment (apparition47) [inline Sources/RLogger/RLogger.swift:37]: Could be more concise without requiring a named parameter (i.e. `verbose(_ message: String)`). Does your test compile? Your test seems to already exclude the named parameter 🤔

Comment (malayli) [reply Sources/RLogger/RLogger.swift:37]: @apparition47  I intentionally wanted a named parameter so there is no ambiguity with Variadicable functions.

This one is based on `@discardableResult static func verbose(_ format: String, arguments: CVarArg...) -> String?`:
RLogger.verbose("test")

Comment (apparition47) [reply Sources/RLogger/RLogger.swift:37]: Ah I missed those function signatures, thanks

**Review by apparition47 — APPROVED** [review]

**Review by donnie-jp — CHANGES_REQUESTED** [review]
See comments on internal PR

**Review by imaginaris — APPROVED** [review]

**Comment by malayli** [comment]
@donnie-jp The changes have been applied. Can you remove your changes request?

**Review by donnie-jp — APPROVED** [review]

File: Sources/RLogger/RLogger.swift [hunk 2]
```diff
@@ -31,7 +31,32 @@ public struct RLogger {
...
+        log(.warning, message: message)
+    }
+
+    @discardableResult
+    public static func error(message: String) -> String? {
+        log(.error, message: message)
+    }
 }
 
 // MARK: - Variadicable
```
Comment (donnie-jp) [inline Sources/RLogger/RLogger.swift:62]: Should the variadics be deprecated here too?

Comment (malayli) [reply Sources/RLogger/RLogger.swift:62]: I think yes as we want RAnalytics SDK to use SDK Utils instead of RLogger right?

**Pull request merged**